### ISSUE TITLE
Find a way to test that Psi4 installations work alongside builds

### ISSUE THREAD

**Issue by mattwthompson** [conda-forge/openff-bespokefit-feedstock#12]
### Comment:

https://github.com/conda-forge/openff-bespokefit-feedstock/pull/11#issuecomment-1414127886

**Comment by j-wags** [comment]
Yeah... Maybe a start a psi4 eval with a 15 second timeout, pipe STDOUT to a file, and grep the file afterwards to make sure it shows some output from after the normal failure point? It'll make our c-f builds a bit slower, but it's way better than putting it in regular CI, especially since the resolution when it fails will be version pins.

**Comment by mattwthompson** [comment]
I now think this could be done in the repo as a standalone test, separate from everything else. Probably a combination of commands like `psi4 --version` and importing `h5py` within Python could safeguard against most issues without needing full QC run time.
